applications/tari_miner/src/config.rs

```diff
@@ -90,8 +90,8 @@ impl SubConfigPath for MinerConfig {
 impl Default for MinerConfig {
     fn default() -> Self {
         Self {
-            base_node_grpc_address: Multiaddr::from_str("/ip4/127.0.0.1/tcp/18142").unwrap(),
-            wallet_grpc_address: Multiaddr::from_str("/ip4/127.0.0.1/tcp/18143").unwrap(),
+            base_node_grpc_address: Multiaddr::from_str("/ip4/127.0.0.1/tcp/18172").unwrap(),
```

Comment:
Bonus points:

```rust
impl Default for MinerConfig {
    fn default() -> Self {
        let default_base_node_port = grpc_default_port(ApplicationType::BaseNode, Network::default());
        let default_wallet_port = grpc_default_port(ApplicationType::ConsoleWallet, Network::default());
        Self {
            base_node_grpc_address: format!("/ip4/127.0.0.1/tcp/{}", default_base_node_port)
                .parse()
                .unwrap(),
            wallet_grpc_address: format!("/ip4/127.0.0.1/tcp/{}", default_wallet_port).parse().unwrap(),
            wallet_grpc_authentication: GrpcAuthentication::default(),
            num_mining_threads: num_cpus::get(),
            mine_on_tip_only: true,
            proof_of_work_algo: ProofOfWork::Sha3,
            validate_tip_timeout_sec: 30,
            mining_pool_address: String::new(),
            mining_wallet_address: String::new(),
            mining_worker_name: String::new(),
            coinbase_extra: "tari_miner".to_string(),
        }
    }
}
```

Reply:
haha yeah this is much better, I did not even think about linking the default in.